src/components/calcite-stepper/calcite-stepper.tsx

```diff
@@ -216,18 +217,28 @@ export class CalciteStepper {
   //
   //--------------------------------------------------------------------------
 
-  private emitChangedItem() {
+  private addHorizontalContentContainer(): void {
+    const contentContainer = document.createElement("div") as HTMLDivElement;
+    contentContainer.classList.add("calcite-stepper-content");
+    // handle ie styles
+    const isIE = !!(navigator.userAgent.match(/Trident/) && !navigator.userAgent.match(/MSIE/));
+    if (isIE) contentContainer.style.cssText = IESTYLES;
+    this.el.insertAdjacentElement("beforeend", contentContainer);
+    this.stepperContentContainer = this.el.querySelector(".calcite-stepper-content");
```

Comment:
☝️ Rather than do a querySelector call here, you should be able to just replace `const contentContainer` with `this.stepperContentContainer = document.createElement("div")`

Reply:
👍 will update.